packages/@aws-cdk/aws-lambda-event-sources/test/integ.kinesiswithdlq.ts

```diff
@@ -0,0 +1,64 @@
+import * as kinesis from '@aws-cdk/aws-kinesis';
+import * as lambda from '@aws-cdk/aws-lambda';
+import * as sqs from '@aws-cdk/aws-sqs';
+import { App, Stack } from "@aws-cdk/core";
+import { KinesisEventSource } from '../lib';
+
+/*
+ * Stack verification steps:
+ * * aws lambda invoke --function-name <deployed 'FP' function name> --invocation-type Event --payload '"Event"' response.json
+ * * Validate two executions of fn lambda
+ * * Validate one message in dlq
```

Comment:
Would be very useful if we could provide how to validate these as AWS CLI commands. You could print the SQS queue ARN or URL that's needed for the CLI as a `CfnOutput`.

Take a look at how this is done [here](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-apigateway/test/authorizers/integ.token-authorizer.ts#L6) and [here](https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-lambda-destinations/test/integ.destinations.ts#L7)

Reply:
I'll give it a crack, but I don't have CLI access so it will be mostly flying blind.

Reply:
What do you mean? 
The [AWS CLI](https://aws.amazon.com/cli/) is available to every one, and I assume you have an AWS account to test this on.